cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceAction.scss

```diff
@@ -123,12 +130,17 @@ $link-color: #0275d8;
                 overflow: scroll;
```

Comment:
This should be `auto`. The current property results in a placeholder for scrollbar even if its not there.

<img width="889" alt="screen shot 2017-01-30 at 11 42 10 am" src="https://cloud.githubusercontent.com/assets/1452845/22438540/395cebaa-e6e1-11e6-87cd-5e14082a73fa.png">